packages/sdk-react/src/core/queries/useCounterpart.ts

```diff
@@ -296,26 +296,129 @@ export const useDeleteCounterpartVat = (counterpartId: string) => {
   });
 };
 
+export interface GenericCounterpartContact {
+  id: string;
+  counterpart_id: string;
+  /**
+   * @description Is default contact person
+   */
+  is_default?: boolean;
+  /** @description The address of a contact person. */
+  address?: components['schemas']['CounterpartAddress'];
+  /**
+   * Format: email
+   * @description The email address of a contact person.
+   * @example contact@example.org
+   */
+  email?: string;
+  /**
+   * @description The person's first name.
+   * @example Adnan
+   */
+  first_name: string;
+  /** @description Indicates if the counterpart is a customer. */
+  is_customer: boolean;
+  /** @description Indicates if the counterpart is a vendor. */
+  is_vendor: boolean;
+  /**
+   * @description The person's last name.
+   * @example Singh
+   */
+  last_name: string;
+  /**
+   * @description The person's phone number.
+   * @example 5553211234
+   */
+  phone?: string;
+  /**
+   * @description The person's title or honorific. Examples: Mr., Ms., Dr., Prof.
+   * @example Mr.
+   */
+  title?: string;
+}
+
 export const useCounterpartContactList = (
   counterpartId: string | undefined
-) => {
+): {
+  data?: GenericCounterpartContact[];
+  error?:
+    | Error
+    | { error: { message: string } }
+    | {
+        detail?:
+          | { loc: (string | number)[]; msg: string; type: string }[]
+          | undefined;
+      }
+    | null;
+  isLoading: boolean;
+} => {
   const { api } = useMoniteContext();
 
-  const { data: counterpart } = api.counterparts.getCounterpartsId.useQuery(
-    { path: { counterpart_id: counterpartId ?? '' } },
-    {
-      enabled: !!counterpartId,
-    }
-  );
-
-  return api.counterparts.getCounterpartsIdContacts.useQuery(
+  const { data: counterpart, isLoading: isCounterpartLoading } =
+    api.counterparts.getCounterpartsId.useQuery(
+      { path: { counterpart_id: counterpartId ?? '' } },
+      {
+        enabled: !!counterpartId,
+      }
+    );
+
+  const {
+    data: contacts,
+    isLoading: areContactsLoading,
+    error,
+  } = api.counterparts.getCounterpartsIdContacts.useQuery(
     {
       path: { counterpart_id: counterpartId ?? '' },
     },
     {
       enabled: Boolean(counterpartId && counterpart?.type === 'organization'),
     }
   );
+
+  if (counterpart?.type === 'individual') {
+    const individual = (
+      counterpart as components['schemas']['CounterpartIndividualRootResponse']
```

Comment:
We have type guard functions that allow us to prevent using type assertions. 

Please see :

```ts
export const isIndividualCounterpart = (
  counterpart: CounterpartResponse
): counterpart is components['schemas']['CounterpartIndividualRootResponse'] =>
  counterpart.type === 'individual';
```
`packages/sdk-react/src/components/counterparts/helpers.ts`